docs/style.md

```diff
@@ -393,3 +393,22 @@ The `INFO` level is enabled by default, use it for information useful for node
 operators. The `DEBUG` level is enabled on the canary nodes, use it for
 information useful in debugging testnet failures. The `TRACE` level is not
 generally enabled, use it for arbitrary debug output.
+
+## Metrics
+
+Consider adding metrics to new functionality. For example, how often each type
+of error was triggered, how often each message type was processed.
+
+**Rationale:** Metrics are cheap to increment, and they often provide a significant
+insight into operation of the code, almost as much as logging. But unlike logging
+metrics don't incur a significant runtime cost.
+
+### Naming
+
+Prefix all `nearcore` metrics with `near_`.
+Follow [https://prometheus.io/docs/practices/naming/](Prometheus naming convention)
```

Comment:
:+1: hugely helpful link, thanks! This is exactly the example of a context which might be obvious to an author of PR/issue, but not to a clueless reviewer, something I've mentioned at all:hand:s